packages/mask/background/services/identity/profile/query.ts

```diff
@@ -28,3 +36,27 @@ export async function mobile_queryProfileRecordFromIndexedDB() {
     if (process.env.architecture !== 'app') throw new TypeError('This function is only available in app')
     return queryProfilesFromIndexedDB({})
 }
+
+export async function queryProfilesInformation(identifiers: ProfileIdentifier[]): Promise<ProfileInformation[]> {
```

Comment:
Could we pass id as sufficient condition instead, so that we don't need to construct a `ProfileIdentifier`.